package.json

```
@@ -180,6 +180,18 @@
                             "type": "string",
                             "description": "Output format for Kubernetes specs. One of 'json' or 'yaml' (default)."
                         },
+                        "vs-kubernetes.nodejs-autodetect-remote-root": {
+                            "type": "boolean",
+                            "description": "If true will try to automatically get the root location of the source code in the container."
+                        },
+                        "vs-kubernetes.nodejs-remote-root": {
+                            "type": "string",
+                            "description": "User specified root location of the source code in the container."
```

Do we need 'user specified' here?  Seems like config entries are inherently 'user specified.'  Also should mentioned Node and debugging.  Maybe e.g.:
```suggestion
                            "description": "The root at which to find Node.JS source code in the container when debugging."
```

---

I wrote User specified as opposed to automatically detected. If you still think this is misleading, I will change it.

---

Yes, I would still like this to be changed please - thanks!